README.md

```diff
@@ -38,12 +38,153 @@ Thank you for considering and taking the time to contribute! Before contributing
 
 ### ⚡ 3. Skill set :
 
-1. Knowledge of git & github.
+1. [Knowledge of git & github.](https://www.youtube.com/watch?v=4iDQ7CHHZZM)
 2. JavaScript
 3. [ReactJS](https://reactjs.org/)
 4. Mobile development Experience in Any Tech (optionally)
+5. Ionic / Capacitor (optionally)
+
+## 📌 Setting up Locale Environment
+
+### 🚩 Forking repository :
+
+1. Fistly you have to make your own copy of project. For that you have to fork the repository. You can find the fork button on the top-right side of the browser window. (Refer image below )
+2. Kindly wait till it gets forked.
+3. After that copy will look like `<your-user-name>/edu-client` forked from `codeforcause/edu-client`.
+   <img src="./doc/images/fork.png" alt="Demo fork image" align="center"/>
+
+### 🚩 Clone repository :
+
+1. Now you have your own copy of project. Here you have to start your work.
+2. Go to desired location on your computer where you want to set-up the project.
+3. Right click there and click on `git bash`. A terminal window will pop up
+4. Type the command `git clone <your-fork-url>.git` and hit enter.(Refer the image to copy url)
+5. Wait for few seconds till the project gets copied
+   <img src="./doc/images/copy.png" alt="Demo copy image" align="center"/>
+
+### 🚩 Setting up remote :
+
+1. Now you have to set up remote repositories
```

Comment:
These things are already in contributing.md https://github.com/codeforcauseorg/edu-client/blob/main/CONTRIBUTING.md

We may not need the full contrubution guideline here.

Reply:
@Abhishek-kumar09 can u pls check it now!?